src/genericComponents/Dropdown.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Col, Form } from 'react-bootstrap';
+
+const DropdownMenu = (props: any) => {
+  const { onChange, values } = props;
+  //do mapping that returns options
+  return (
+    <Form.Group as={Col} controlId="formGridState">
```

Comment:
You can remove `controlId`, or add an optional prop for it and make it dynamic. `formGridState` is just from the Bootstrap example where the dropdown is for selecting a state, but ours will be used for many purposes (not just selecting a state)